app/views/reports/regions/index.html.erb

```diff
@@ -21,24 +21,6 @@
             </div>
           <% end %>
         </div>
-
-        <script>
-          const search = new RegionsSearch().search;
-          const form = $("#search-form")
-          form.on("submit", function(event) {
-            input = $(event.target).find(".typeahead-input")
-            search(input);
-            return false;
-          })
-          form.on("keyup", function(event) {
-            search(event.target);
-            return false;
-          })
-
-          const async = false
-          new AdminAccess("facility-access").initialize(async)
-        </script>
-
       <% else %>
```

Comment:
I may be missing something, but doesn't the index page now need the search partial as well?

Reply:
This page does render the search partial, above on [line 8](https://github.com/simpledotorg/simple-server/pull/2006/files#diff-73451de3a663ee58032c74f69fb20b378538fd8e388f7b1cf79b1c04b8241ba7R8).